Packs/SecurityScorecard/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "SecurityScorecard",
     "description": "Provides security scorecards and alerts for domains.",
     "support": "partner",
-    "currentVersion": "1.0.7",
+    "currentVersion": "1.1.0",
```

This is only a revision [xsoar.pan.dev](https://xsoar.pan.dev/docs/documentation/release-notes#examples:~:text=add%20many%20commands.-,revision,-When%20you%20add)
```suggestion
    "currentVersion": "1.0.8",
```

---

I would say that 3 commands are only a few :)

---

Ok, fine by me!